### PR TITLE
Update configuration of RSpec cops

### PR DESCRIPTION
**Pull request by ThomasKoppensteiner**

### CODE REVIEW

File: config/default.yml [hunk 1]
```diff
@@ -63,96 +64,133 @@ Metrics/ModuleLength:
...
+# https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecexamplewording
 RSpec/ExampleWording:
   IgnoredWords:
     - only
   CustomTransform:
     have: has
 
-RSpec/FactoryBot/SyntaxMethods:
+# https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecspecfilepathformat
+RSpec/SpecFilePathFormat:
```
Comment (ThomasKoppensteiner) [inline config/default.yml:160]: Enabling it and ignoring `spec/integrations` here could also work.
What do you think?

File: config/default.yml [hunk 2]
```diff
@@ -63,96 +64,133 @@ Metrics/ModuleLength:
...
-RSpec/NamedSubject:
+# https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecmultiplememoizedhelpers
+RSpec/MultipleMemoizedHelpers:
   Enabled: false
 
-RSpec/NestedGroups:
-  Enabled: false
+# https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecnamedsubject
+RSpec/NamedSubject:
+  EnforcedStyle: named_only
```
Comment (ThomasKoppensteiner) [inline config/default.yml:181]: Changed to `named_only`. I think this is makes more sense.

**Review by andreaseger — APPROVED** [review]
lgtm

**Review by TheWudu — APPROVED** [review]

File: config/default.yml (hunk 1 above)
Comment (Goltergaul) [reply config/default.yml:160]: disabling sounds fine to me

**Pull request merged**